editors/vscode/src/client.ts

```diff
@@ -400,41 +400,54 @@ function getGoalsEnvContent(goals : Goal[]){
     let codeHyps : String = ""; //hypothesis HTML code
     let codeGoals : String = ""; //goals HTML code
     let codeEnvGoals : String = ""; //result code HTML
-
+    let goalNo = 0;
+    
     for(let i=0; i < goals.length; i++) {
```

Comment:
goalNo and i are equal. why not using i only?

Reply:
`goalNo` only gets increments for type goals. I changed the variable name to `typGoalNo`